index.php

```diff
@@ -97,10 +97,8 @@
             var layer_tidalscale;                  // 16
             var layer_permalink;                   // 17
             var layer_waterdepth_trackpoints_100m;      // 18
-            var layer_elevation_profile_contours;  // 19
-            var layer_elevation_profile_hillshade; // 20
-            var layer_waterdepth_trackpoints_10m;      // 21
-            var layer_waterdepth_contours;        // 22
+            var layer_waterdepth_trackpoints_10m;      // 19
+            var layer_waterdepth_contours;        // 20
```

Comment:
```suggestion
            var layer_waterdepth_trackpoints_10m;      // 21
            var layer_waterdepth_contours;        // 22
```

Comments show the `layerId` from `OpenLayers.Layer`. As the layer IDs do not change by this pull request (and should not do), the comments need to be unchanged.

Not updating the layer IDs is important to keep old permalinks working.